nixos/modules/services/web-apps/rss-bridge.nix

```
@@ -5,10 +5,18 @@ let
 
   poolName = "rss-bridge";
 
-  whitelist = pkgs.writeText "rss-bridge_whitelist.txt"
-    (concatStringsSep "\n" cfg.whitelist);
+  configAttr = { FileCache.path = "${cfg.dataDir}/cache/"; } // cfg.config;
+  cfgHalf = lib.attrsets.mapAttrsRecursive (path: value: let
+    envName = lib.toUpper ("RSSBRIDGE_" + lib.concatStringsSep "_" path);
+    envValue = if lib.isList value then lib.concatStringsSep "," value else if lib.isBool value then lib.boolToString value else builtins.toString value;
```

```suggestion
    envValue = if lib.isList value then
      lib.concatStringsSep "," value
    else if lib.isBool value then
      lib.boolToString value
    else
      toString value;
```
Lets make this a bit more readable